examples/03-Maxwell/Maxwell2D_Electrostatic.py

```diff
@@ -0,0 +1,196 @@
+It shows how to create the geometry, load material properties from an Excel file and
+set up the mesh settings. Moreover, it focuses on post-processing operations, in particular how to
+plot field line traces, relevant for an electrostatic analysis.
+
+"""
+#################################################################################
+# Perform required imports
+# ~~~~~~~~~~~~~~~~~~~~~~~~
+# Perform required imports.
+
+import pyaedt
+
+#################################################################################
+# Initialize Maxwell 2D
+# ~~~~~~~~~~~~~~~~~~~~~
+# Initialize Maxwell 2D, providing the version, path to the project, and the design
+# name and type.
+
+desktopVersion = '2023.2'
+
+sName = 'MySetupAuto'
+sType = 'Electrostatic'
+dName = 'Design1'
+pName = pyaedt.generate_unique_project_name()
+non_graphical = False
+
+#################################################################################
+# Download .xlsx file
+# ~~~~~~~~~~~~~~~~~~~
+# Set local temporary folder to export the .xlsx file to.
+
+file_name_xlsx = pyaedt.downloads.download_file("field_line_traces", "my_copper.xlsx")
+
+#################################################################################
+# Initialize dictionaries
+# ~~~~~~~~~~~~~~~~~~~~~~~
+# Initialize dictionaries that contain all the definitions for the design variables.
+
+geom_params_circle = {
+    'circle_x0': '-10mm',
+    'circle_y0': '0mm',
+    'circle_z0': '0mm',
+    'circle_axis': 'Z',
+    'circle_radius': '1mm'
+}
+
+geom_params_rectangle = {
+    'r_x0': '1mm',
+    'r_y0': '5mm',
+    'r_z0': '0mm',
+    'r_axis': 'Z',
+    'r_dx': '-1mm',
+    'r_dy': '-10mm'
+}
+
+##################################################################################
+# Launch Maxwell 2D
+# ~~~~~~~~~~~~~~~~~
+# Launch Maxwell 2D and save the project.
+
+M2D = pyaedt.Maxwell2d(projectname=pName,
+                       specified_version=desktopVersion,
+                       designname=dName,
+                       solution_type=sType,
+                       new_desktop_session=True,
+                       non_graphical=non_graphical
+                       )
+
+##################################################################################
+# Create object to access 2D modeler
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Create the object ``mod2D`` to access the 2D modeler easily.
+
+mod2D = M2D.modeler
+mod2D.delete()
+mod2D.model_units = "mm"
+
+##################################################################################
+# Define variables from dictionaries
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Define design variables from the created dictionaries.
+
+for k, v in geom_params_circle.items():
+    M2D[k] = v
+for k, v in geom_params_rectangle.items():
+    M2D[k] = v
+
+##################################################################################
+# Read materials from .xslx file
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Read materials from .xslx file into and set into design.
```

Comment:
```suggestion
# Read materials from the XSLX file into and set into design.
```
Into the what? Or should it be: "# Read materials from the XSLX file and set into design."